iac-extensions/kubernetes/src/test/resources/checks/MemoryLimitCheck/memory_limit_deployment_with_global_limit.yaml

```diff
@@ -0,0 +1,50 @@
+# This file duplicates `memory_limit_deployment.yaml` but adds a namespace to the deployment.
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: example
+  namespace: with-global-limit
+  labels:
+    app: web
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: web
+  template:
+    metadata:
+      labels:
+        app: web
+    spec:
+      containers:
+        - name: noncompliant_without_memory_limit
```

Comment:
Actually the names here and in following lines should be adjusted as they are now compliant